cmd/zc_traverser_blobfs.go

```diff
@@ -127,8 +136,23 @@ func (t *blobFSTraverser) traverse(processor objectProcessor, filters []objectFi
 					*v.ContentLength,
 					v.ContentMD5(),
 					blobTypeNA,
+					bfsURLParts.FileSystemName,
 				)
 
+				/* TODO: Enable this code segment in the case we ever do BlobFS->Blob transfers.
+
```

Comment:
it'll be just blob -> blob at that point, with the interop support

Reply:
Yup

Reply:
I more or less just left it for sanity's sake.